README.md

```diff
@@ -21,6 +21,44 @@ You can find the latest Spark documentation, including a programming
 guide, on the [project web page](https://spark.apache.org/documentation.html).
 This README file only contains basic setup instructions.
 
+# Daily Build Pipeline Status
+
+| Branch     | Topic                                                  | Status                                                                                                                                                                                                          |
+|------------|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| master     | Java 21                                                | [![GitHub Actions Build](https://github.com/apache/spark/actions/workflows/build_java21.yml/badge.svg)](https://github.com/apache/spark/actions/workflows/build_java21.yml)                                     |
```

Comment:
Actually do we need branch and topic? they are shown in the titles.

Reply:
Removed `Topic`, does keeping the `branch` seem more organized?